bot.py

```diff
@@ -90,6 +91,23 @@ async def abort_command(update: Update, context: ContextTypes.DEFAULT_TYPE):
     else:
         await update.message.reply_text("You don't have an active session. Use /on to start a new timer.")
 
+async def hour_command(update: Update, context: ContextTypes.DEFAULT_TYPE):
+    user_name = update.effective_user.username
+    if USER_NAME_MAPPING.get(user_name, "Unknown") == "Unknown":
```

Comment:
Standardize to user_name not in USER_NAME_MAPPING

Reply:
[Resolved!](https://github.com/samtjong23/airconTeleBot/pull/3/commits/acd25ca16c9fa5596825f13fb850daca1873e381)